packages/@aws-cdk/aws-codepipeline/lib/private/cross-region-support-stack.ts

```diff
@@ -53,11 +60,13 @@ export class CrossRegionSupportConstruct extends Construct {
     super(scope, id);
 
     const createKmsKey = props.createKmsKey ?? true;
+    const enableKeyRotation = props.enableKeyRotation ?? false;
```

Comment:
Same here - we should leave this as `undefined` to prevent changes to existing CloudFormation templates. Given that, I would get rid of this local variable completely.

```suggestion
```